src/Bicep.Core.UnitTests/Configuration/ConfigurationManagerTests.cs

```diff
@@ -544,7 +547,8 @@ public void GetConfiguration_ValidCustomConfiguration_OverridesBuiltInConfigurat
     ""imports"": true,
     ""resourceTypedParamsAndOutputs"": null,
     ""sourceMapping"": null,
-    ""paramsFiles"": null
+    ""paramsFiles"": null,
+    ""aggregateTypes"": null
```

Comment:
`aggregateTypes` doesn't feel super intuitive to me - do you have any other ideas for naming? What about something like `customTypes`, `userDefinedTypes`, `complexTypes`?

Reply:
I went with `userDefinedTypes` in the latest batch of changes.